Makefile

```diff
@@ -3,6 +3,26 @@ GO := go
 all : build
```

Comment:
Can you add `gofmt -s ./cmd/mailchain/internal/http/handlers/openapi.go` to `Makefile` 

```

echo '`' >>  ./cmd/mailchain/internal/http/handlers/openapi.go
	echo '}' >>  ./cmd/mailchain/internal/http/handlers/openapi.go
ADD HERE
	addlicense -l apache -c Finobo ./cmd/mailchain/internal/http/handlers/openapi.go	
	rm -rf vendor
```
Then this will work

Reply:
Done [x]